utils/overwrite_expected_slice.py

```diff
@@ -76,7 +76,7 @@ def main(correct, fail=None):
 
     done_tests = defaultdict(int)
     for line in correct_lines:
-        file, class_name, test_name, correct_line = line.split(";")
+        file, class_name, test_name, correct_line = line.split("::")
```

Comment:
Fix a bug in this script when identifying the test modules where slices need to be replaced.